### PR TITLE
ADD css & assets watch mode for rebundle & livereload. Fixes atomify-css#24

### PR DESCRIPTION
**Pull request by serapath**

### CODE REVIEW

**Comment by serapath** [comment]
So it took me a bit more than "some weeks". I'll already updated pull requests to:
`rework-assets` and `atomify` and will no work on `atomify-css`.
Maybe you could give some feedback on the changed source?

File: package.json [hunk 1]
```diff
@@ -24,17 +24,18 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "bower-less": "^0.0.1",
+    "bower-less": "0.0.1",
```
Comment (joeybaker) [inline package.json]: Why is it necessary to pin this version?

File: package.json [hunk 2]
```diff
@@ -24,17 +24,18 @@
...
     "mkdirp": "^0.5.0",
     "npm-less": "^1.0.0",
     "package-lookup": "^0.1.2",
-    "resolve": "^1.0.0",
+    "resolve": "^1.1.0",
     "resrcify": "^1.0.1",
     "rework": "^1.0.0",
     "rework-assets": "^1.0.0",
-    "rework-bower": "^0.0.1",
+    "rework-bower": "0.0.1",
```
Comment (joeybaker) [inline package.json]: Why pin this?

File: package.json [hunk 3]
```diff
@@ -24,17 +24,18 @@
...
     "rework-npm": "^1.0.0",
     "rework-vars": "^3.1.1",
+    "sane-scaffold": "^1.0.5",
```
Comment (joeybaker) [inline package.json]: Where do we use this?

File: css.js [hunk 4]
```diff
@@ -66,6 +67,9 @@ function applyReworkAssets (css, opts, dirName) {
             , dest: opts.assets.dest || ''
             , prefix: opts.assets.prefix || ''
             , retainName: opts.assets.retainName || ''
+            , onFile: function onFile(filename) {
+              ressourcepaths.assetfiles.push(filename);
```
Comment (joeybaker) [inline css.js]: nit pick, but: no semicolons :)

File: package.json (hunk 1 above)
Comment (serapath) [inline package.json]: i did not do that and i dont know how that happened.
I will revert it. ...strange

File: package.json (hunk 2 above)
Comment (serapath) [inline package.json]: i did not do that and i dont know how that happened.
I will revert it. ...strange

File: package.json (hunk 3 above)
Comment (serapath) [inline package.json]: i did not do that and i dont know how that happened.
I will revert it. ...strange

File: css.js (hunk 4 above)
Comment (serapath) [inline css.js]: ah damn, sorry, yes i will remove semicolons.

File: package.json (hunk 3 above)
Comment (serapath) [inline package.json]: ok, done!

**Comment by joeybaker** [comment]
This seems fine to merge. @serapath can you just confirm that this won't break current functionality? It doesn't appear to.

**Comment by serapath** [comment]
It should not. I'm using it for quite some time now and haven't run into any issues in the last weeks, or is it already months :-)

**Pull request merged**